tutor/admin.py

```diff
@@ -3,4 +3,4 @@
 the model, the data will be accessible through Django's admin functionality.
 """
 
-# Register your models here.
+# Register your models here.
```

Comment:
There needs to be a newline after this comment. Also, make sure that you use Unix-style line endings (`LF`) and not Windows (`CRLF`). If you are using PyCharm, the bottom row of the main window should allow you to change from Windows style to Unix style.

Reply:
I almost did that, but when I looked at the code here it didn't appear to have a new line. PyCharm says it's using LF. Is there something else I need to do?